Makefile

```diff
@@ -91,7 +91,42 @@ build: build-server build-nightly-server build-ad-server build-client \
 .PHONY: build
 
 
-
+.PHONY: debug-vars
+debug-vars:
+	@echo OS_NAME: $(OS_NAME)
+	@echo OS_PREFIX: $(OS_PREFIX)
+	@echo TAG: $(TAG)
+	@echo NIGHTLY_TAG: $(NIGHTLY_TAG)
+	@echo SERVER_NAME: $(SERVER_NAME)
+	@echo SERVER_REPO_NAME: $(SERVER_REPO_NAME)
+	@echo NIGHTLY_SERVER_REPO_NAME: $(NIGHTLY_SERVER_REPO_NAME)
+	@echo NIGHTLY_SERVER_NAME: $(NIGHTLY_SERVER_NAME)
+	@echo AD_SERVER_NAME: $(AD_SERVER_NAME)
+	@echo AD_SERVER_REPO_NAME: $(AD_SERVER_REPO_NAME)
+	@echo NIGHTLY_AD_SERVER_NAME: $(NIGHTLY_AD_SERVER_NAME)
+	@echo NIGHTLY_AD_SERVER_NAME: $(NIGHTLY_AD_SERVER_NAME)
+	@echo NIGHTLY_AD_SERVER_REPO_NAME: $(NIGHTLY_AD_SERVER_REPO_NAME)
+	@echo CLIENT_NAME: $(CLIENT_NAME)
+	@echo CLIENT_REPO_NAME: $(CLIENT_REPO_NAME)
+	@echo NIGHTLY_CLIENT_NAME: $(NIGHTLY_CLIENT_NAME)
+	@echo NIGHTLY_CLIENT_REPO_NAME: $(NIGHTLY_CLIENT_REPO_NAME)
+	@echo TOOLBOX_NAME: $(TOOLBOX_NAME)
+	@echo TOOLBOX_REPO_NAME: $(TOOLBOX_REPO_NAME)
+	@echo NIGHTLY_TOOLBOX_NAME: $(NIGHTLY_TOOLBOX_NAME)
+	@echo NIGHTLY_TOOLBOX_REPO_NAME: $(NIGHTLY_TOOLBOX_REPO_NAME)
+
+	@echo BUILDFILE_SERVER: $(BUILDFILE_SERVER)
+	@echo BUILDFILE_AD_SERVER: $(BUILDFILE_AD_SERVER)
+	@echo BUILDFILE_NIGHTLY_AD_SERVER: $(BUILDFILE_iNIGHTLY_AD_SERVER)
+	@echo BUILDFILE_NIGHTLY_SERVER: $(BUILDFILE_NIGHTLY_SERVER)
+	@echo BUILDFILE_CLIENT: $(BUILDFILE_CLIENT)
+	@echo BUILDFILE_TOOLBOX: $(BUILDFILE_TOOLBOX)
+	@echo BUILDFILE_NIGHTLY_TOOLBOX: $(BUILDFILE_NIGHTLY_TOOLBOX)
+
+	@echo SERVER_SRC_FILE: $(SERVER_SRC_FILE)
+	@echo AD_SERVER_SRC_FILE: $(AD_SERVER_SRC_FILE)
+	@echo CLIENT_SERVER_SRC_FILE: $(CLIENT_SRC_FILE)
+	@echo TOOLBOX_SRC_FILE: $(TOOLBOX_SRC_FILE)
```

Comment:
I prefer using the built-in [info](https://www.gnu.org/software/make/manual/make.html#index-info) function for such printing. It is just a matter of style -- no changes need here.

Reply:
@synarete  wrote:
> I prefer using the built-in [info]
(https://www.gnu.org/software/make/manual/make.html#index-info) function for such printing. It is just a matter of style -- no changes need here.

interesting idea again! I will also look at this later, separately. ...